frigate/events/cleanup.py

```diff
@@ -257,7 +257,11 @@ def expire_clips(self) -> list[str]:
         events_to_update = []
 
         for batch in query.iterator():
-            events_to_update.extend([event.id for event in batch])
+            try:
+                events_to_update.extend([event.id for event in batch])
```

Comment:
Doesn't `.iterator()` always return a single record? So `batch` will never be more than 1 record?

Reply:
oh yeah, not sure how the logic got mixed up